src/Compilers/Lua/Portable/Syntax/SyntaxKind.cs

```diff
@@ -533,6 +539,7 @@ public enum SyntaxKind : ushort
         IdentifierName = 2020,
         IfExpression = 2080,
         ElseIfExpressionClause = 2081,
+        HashStringLiteralExpression = 2082,
```

Comment:
Same as above, just update the nodes gap comment to reflect the new free range please